components/03-sections/02-navigation/04-side-navigation-news/side-navigation-news.hbs

```diff
@@ -0,0 +1,87 @@
+<!-- Side Navigation -->
+<div class="sidebar-news">
+    {{!-- Related Stories Links --}}
+    <div class="sidebar-news-container">  
+        <div class="sidebar-news-title">Related News</div>
+            <ul class="related-news-items">
+                <li class="related-news-item"> <span class="related-news-link-icon"></span> <span class="related-news-link"><a href="#" aria-label="Related News Link">Link to another related article or promoted news article a really long title for this one.</a></span></li>
```

Comment:
We could remove the `span` tags and the `li` class tags to reduce confusion, since you might be able to target items without them.

Reply:
I disagree. You need the spans to get the hanging indent for the icons. Unless you know a better way of doing that.